docs/containers-registries.conf.5.md

```diff
@@ -104,20 +104,32 @@ contacted and contains the image will be used (and if none of the mirrors contai
 the primary location specified by the `registry.location` field, or using the unmodified
 user-specified reference, is tried last).
 
-Each TOML table in the `mirror` array can contain the following fields, with the same semantics
-as if specified in the `[[registry]]` TOML table directly:
-- `location`
-- `insecure`
+Each TOML table in the `mirror` array can contain the following fields:
+- `location`： same semantics
+as specified in the `[[registry]]` TOML table
+- `insecure`： same semantics
+as specified in the `[[registry]]` TOML table
+- `digest-only`： `true` or `false`. If `true` an individual mirror will only be used during pulling if the image reference includes a digest.
+- `tag-only`： `true` or `false`. If `true` an individual mirror will only be used during pulling if the image reference includes a tag.
+Note that digest-only will be overridden if mirror-by-digest-only is set for the entire registry, tag-only will not be overridden.
+tag-only will be overridden if mirror-by-tag-only is set for the entire registry, digest-only will not be overridden.
```

Comment:
Quickly reading this, it’s unclear to me what is the resulting behavior of “will (not) be overridden”.

Especially the way how a mirror can “opt out” of the registry-level $something-only by setting $opposite-only is to me, completely non-obvious from this text.

I’m not even sure we need to define these things — alternatively we could forbid mirror-level *-only options if the registry-level *-only is set, and thus allow the administrator to either define registry-level, or mirror-level, settings, and avoid these conflicts. (@vrothberg ?)

Reply:
> and thus allow the administrator to either define registry-level, or mirror-level, settings, and avoid these conflicts

That sounds like a clean model to me.